app/controllers/contacts/new.js

```diff
@@ -174,7 +176,13 @@ export default class ContactsNewController extends Controller {
       contact.modified = newDate;
       return await contact.save();
     }));
-    this.transitionToRoute('contacts.overview');
+
+    // this.contacts.forEach(contact => contactItems.addObject(contact));
+
+    // yield contactItems.save();
+    // yield this.model.save();
+    //zonder contact toe te voegen aan model en daarna nog eens model op te slagen, worden de contacten niet gelinked aan model
```

Comment:
Dit is niet nodig omdat op hierboven `contact.contactList = this.model` al gedaan wordt en de relatie zo opgeslagen wordt.